exercises/practice/zipper/.meta/config.json

```
@@ -0,0 +1,10 @@
+{
+  "authors" : [ "porkostomus" ],
+  "contributors" : [ ],
+  "files" : {
+    "solution" : [ "src/zipper.clj" ],
+    "test" : [ "test/zipper_test.clj" ],
+    "example" : [ ".meta/src/example.clj" ]
+  },
+  "blurb" : "\"Creating a zipper for a binary tree.\""
```

Did `configlet` add the quotes?

```suggestion
  "blurb" : "Creating a zipper for a binary tree."
```